arviz/tests/base_tests/test_plots_bokeh.py

```diff
@@ -966,3 +967,18 @@ def test_plot_rank(models, kwargs):
 def test_plot_dist_comparison_warn(models):
     with pytest.raises(NotImplementedError, match="The bokeh backend.+Use matplotlib bakend."):
         plot_dist_comparison(models.model_1, backend="bokeh")
+
+
```

Comment:
I think everything will work with multidimensional data (that is `chain, draw, *shape` with `shape` being multidimensional) thanks to the skip_dims and careful reshaping, however, it would be great to test on multidimensional models too. Should be the same as test below but using `multidim_models` fixture instead, probably a couple cases are enough for multidim.